src/css/fruits.css

```diff
@@ -0,0 +1,321 @@
+  line-height: 129%;
+  letter-spacing: -0.04em;
+  color: #ae1332;
+  background-color: #ffffff;
+  text-align: left;
+  display: flex;
+  border: none;
+  border-radius: 200px;
+  padding: 12px 24px;
+  width: 126px;
+  height: 42px;
+  margin: 0px 0px 24px 24px;
+}
+
+.fruits__container .fruits__order:hover,
+.fruits__container .fruits__order:active {
+  color: #9dc262;
+  background-color: #ffffff;
+  fill: #9dc262;
+}
+
+.fruits__order {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding: 0;
+  gap: 4px;
+  align-items: center;
+  margin-left: auto;
+  margin-right: auto;
+  width: 80px;
+  height: 80px;
+  border-radius: 100%;
+  border: none;
+  fill: #ffffff;
+  color: #ffffff;
+  background-color: #9dc262;
+  letter-spacing: 0.04em;
+  font-family: 'DM Sans', sans-serif;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 1.29;
+  text-transform: uppercase;
+  text-shadow: 0px 4px 70px rgba(0, 0, 0, 0.1);
+  cursor: pointer;
+  -webkit-animation: jello-horizontal 3s both;
+  animation: jello-horizontal 3s both;
+  animation-iteration-count: infinite;
+}
+
+@keyframes jello-horizontal {
+  50% {
+    -webkit-transform: scale3d(1, 1, 1);
+    transform: scale3d(1, 1, 1);
+  }
+  58% {
+    -webkit-transform: scale3d(1.25, 0.75, 1);
+    transform: scale3d(1.25, 0.75, 1);
+  }
+  62% {
+    -webkit-transform: scale3d(0.75, 1.25, 1);
+    transform: scale3d(0.75, 1.25, 1);
+  }
+  70% {
+    -webkit-transform: scale3d(1.15, 0.85, 1);
+    transform: scale3d(1.15, 0.85, 1);
+  }
+  75% {
+    -webkit-transform: scale3d(0.95, 1.05, 1);
+    transform: scale3d(0.95, 1.05, 1);
+  }
+  80% {
+    -webkit-transform: scale3d(1.05, 0.95, 1);
+    transform: scale3d(1.05, 0.95, 1);
+  }
+  100% {
+    -webkit-transform: scale3d(1, 1, 1);
+    transform: scale3d(1, 1, 1);
+  }
+}
+
+.fruits__order_icon {
+  position: relative;
+  top: 10%;
+}
+
+.fruits__order_transform {
+  transform: rotate(-14deg);
+}
+
+/* responsywne style */
+
+/* tablet */
+
+@media screen and (min-width: 768px) {
+  .fruits__container {
+    max-width: 768px;
+  }
+
+  .fruits__first {
+    text-align: center;
+  }
+
+  .fruits__title {
+    font-size: 40px;
+    line-height: 110%;
+  }
+
+  .fruits__plums {
+    display: block;
+  }
+
+  .fruits__orange {
+    display: block;
+  }
+
+  .fruits__text {
+    display: inline-block;
+    max-width: 547px;
+  }
+
+  .fruits__list {
+    gap: 14px;
+  }
+
+  .fruits__item {
+    width: 344px;
+  }
+
+  .fruits__subtitle {
+    font-size: 32px;
+    line-height: 119%;
+  }
+
+  .fruits__description {
+    font-size: 18px;
+    line-height: 133%;
+  }
+
+  .fruits-item-description {
+    padding: 64px 0px 5px 24px;
+  }
+
+  .fruits__img {
+    padding: 0 24px 4px 25px;
+  }
+
+  .fruits__price {
+    font-size: 18px;
+    line-height: 133%;
+    padding: 12px 18px;
+    width: 136px;
+    height: 48px;
+    justify-content: center;
+  }
+
+  .fruits__order {
+    font-size: 18px;
```

Comment:
Sprawdź proszę parametry z figmą. I gdzie masz ułożenie przycisków ? position, i umiejscowienie?

CZEKAJ. tu są inne parametry niż VSC. SPrawdź to proszę zeby bylo aktualne tu i tu.